### PR TITLE
Fix bug in loadBudget

### PR DESCRIPTION
**Pull request by ryan1604**

### CODE REVIEW

**Review by NeoMinWei — APPROVED** [review]
LGTM.
Very prompt bug fixing.
Excellent 💯💯💯

**Review by wwweert123 — APPROVED** [review]
Thank you

File: src/main/java/seedu/financialplanner/storage/LoadData.java [hunk 1]
```diff
@@ -65,6 +65,9 @@ private static void handleCorruptedFile(String message) throws FinancialPlannerE
     private static void loadBudget(String[] split) throws IllegalArgumentException {
         double initial = Double.parseDouble(split[1].trim());
         double current = Double.parseDouble(split[2].trim());
+        if (initial == 0 && current == 0) {
+            return;
+        }
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/storage/LoadData.java:70]: very prompt bug fix, excellent work ethic

File: src/main/java/seedu/financialplanner/storage/LoadData.java [hunk 2]
```diff
@@ -65,6 +65,9 @@ private static void handleCorruptedFile(String message) throws FinancialPlannerE
     private static void loadBudget(String[] split) throws IllegalArgumentException {
         double initial = Double.parseDouble(split[1].trim());
         double current = Double.parseDouble(split[2].trim());
+        if (initial == 0 && current == 0) {
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/storage/LoadData.java:68]: very prompt bug fix, excellent work ethic

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**